chef/cookbooks/ceph/providers/client.rb

```diff
@@ -72,3 +84,11 @@ def auth_set_key(keyname, caps)
   end
   get_or_create.error!
 end
+
+def create_pool(pool_name, pg_num)
+  Chef::Log.info("Creating ceph pool '#{pool_name}' '#{pg_num}'")
```

Comment:
IMHO pg_num should be an optional parameter here, and it should default to something sensible (depending on the pool size ?)